helmfile/values/grafana-user.yaml.gotmpl

```diff
@@ -2,7 +2,7 @@ adminPassword: {{ .Values.user.grafanaPassword }}
 
 image:
   repository: grafana/grafana
-  tag: 7.0.3
+  tag: 7.3.3
```

Comment:
I'd remove this as it is default in the chart https://github.com/grafana/helm-charts/blob/grafana-6.1.11/charts/grafana/values.yaml#L56.
One less tag that we have to worry about updating.

Reply:
cool, will do